src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -138511,7 +138566,14 @@ class B2 : A2
     public override void F6<T>([DisallowNull] T t) { }
 }";
             comp = CreateCompilation(new[] { sourceB2, DisallowNullAttributeDefinition, NotNullAttributeDefinition }, references: new[] { refA });
-            comp.VerifyDiagnostics();
+            comp.VerifyDiagnostics(
+                // (14,26): warning CS8765: Nullability of type of parameter 't' doesn't match overridden member (possibly because of nullability attributes).
```

Comment:
>// (14,26): warning CS8765: Nullability of type of parameter 't' doesn't match overridden member (possibly because of nullability attributes). [](start = 16, length = 142)

Is this warning caused by the attribute? Could we add an override without it to observe the fact?